06-building-a-program-start-to-finish/automating-tasks/output/README.md

```diff
@@ -0,0 +1,6 @@
+#### OUTPUT
+- Link to Screencast on YouTube
+	-- [Browser watching and reloading with gulp](https://youtu.be/UK4qduIZT3k)
```

Comment:
Nice work!

I however don't see you implement the reload for modified JS files.